src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents.mdx

```diff
@@ -26,14 +26,14 @@ import alertsByCondition from 'images/alerts_diagram_by-condition.webp'
 import alertsBySignal from 'images/alerts_diagram_by-signal.webp'
 
 <Callout variant="important">
-  In this document, we use the new terms "issue" and "incident." Please see [this article](https://discuss.newrelic.com/t/new-terminology-coming-for-alerts/163727) for a full explanation of these new terms.
+  In this document, we use the new terms "issue" and "incident." Please see [this article](https://discuss.newrelic.com/t/new-terminology-coming-for-alerts/163727) for a full explanation of these terms.
 </Callout>
 
 An issue is a collection of one or more incidents.
 
-Issues determine when you are notified about incidents disrupting your business. These incidents occur when your given alert conditions (like CPU overage, synthetic monitoring crashes, or slow web app response times) are met.
+Issues determine when you're notified about incidents disrupting your business. These incidents occur when your given alert conditions (like CPU overage, synthetic monitoring crashes, or slow web app response times) are met.
 
-Control which policy conditions create issues. This ensures that you are notified about important ongoing or recurring problems and not bombarded by unnecessary notifications.
+When you can control the policy conditions that create issues, you make sure that you're notified about important ongoing or recurring problems. This way you won't be bombarded with unnecessary notifications and avoid noise.
```

Comment:
Similar to the comment above-- I think there needs to be a deeper explanation of policies and conditions at the beginning of the document.